deeptools/heatmapper.py

```diff
@@ -763,21 +781,18 @@ def getRegionsAndGroups(regions_file, onlyMultiplesOf=1,
                 continue
 
             if ginterval.line.startswith('#'):
-                if includedintervals > 1 and  \
-                        includedintervals - group_boundaries[-1] > 1:
-                    label = ginterval.line[1:].strip()
-                    newlabel = label
-                    if label in group_labels:
-                       # loop to find a unique label name
-                        i = 0
-                        while True:
-                            i += 1
-                            newlabel = label + "_r" + str(i)
-                            if newlabel not in group_labels:
-                                break
-
-                    group_labels.append(label)
-                    group_boundaries.append(includedintervals)
+                group_idx += 1
+                label = ginterval.line[1:].strip()
```

Comment:
This is ok. The label could have white spaces either left or right, so strip() is the right choice. It is line[1:], because at position zero is the '#' that labels the groups.